src/base_consist.h

```diff
@@ -18,8 +18,8 @@ struct BaseConsist {
 	std::string name;                   ///< Name of vehicle
 
 	/* Used for timetabling. */
-	uint32_t current_order_time;               ///< How many ticks have passed since this order started.
-	int32_t lateness_counter;                  ///< How many ticks late (or early if negative) this vehicle is.
+	TimerGameTick::Ticks current_order_time;    ///< How many ticks have passed since this order started.
```

Comment:
This is saved as uint32, but you've changed it to an int32 instead. The data type is the same size so it'll appear to be fine, but some changes for saveload will be necessary.